src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppPlugin.scala

```diff
@@ -194,3 +196,44 @@ object SbtReactiveAppPlugin extends AutoPlugin {
       .fromInputStream(getClass.getClassLoader.getResourceAsStream(name))
       .mkString
 }
+
+object SbtReactiveAppLagomScalaPlugin extends AutoPlugin {
+  private val classLoader = this.getClass.getClassLoader
+
+  override def requires = magic.Lagom.lagomScalaPlugin(classLoader) match {
+    case Success(plugin) => SbtReactiveAppPlugin && plugin
+    case Failure(_) => empty
```

Comment:
Just wondering  what are the semantics of `empty`? Does that mean require something that will never be loaded, i.e. this plugin will never be loaded if `lagomScalaPlugin` returns `Failure`?

Reply:
@longshorej - I think I inadvertently did the wrong thing - `empty` actually does the opposite of what I wanted - the plugin will aways be loaded since the `empty` refers to the following:

```
  private[sbt] final object Empty extends Plugins {
    def &&(o: Basic): Plugins = o
    override def toString = "<none>"
  }
```

I'll fix this shortly.